js/bitmart.js

```diff
@@ -2771,6 +2772,96 @@ module.exports = class bitmart extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        /**
+         * @method
+         * @name bitmart#fetchBorrowRates
+         * @description fetch the borrow interest rates of all currencies
+         * @see https://developer-pro.bitmart.com/en/spot/#get-trading-pair-borrowing-rate-and-amount
+         * @param {object} params extra parameters specific to the bitmart api endpoint
+         * @returns {object} a list of [borrow rate structures]{@link https://docs.ccxt.com/en/latest/manual.html#borrow-rate-structure}
+         */
+        await this.loadMarkets ();
+        const request = {};
+        const response = await this.privateGetSpotV1MarginIsolatedPairs (this.extend (request, params));
```

Comment:
@Dan-krm We don't need to instantiate `request` here since it's empty we can just do this:

```Javascript
 const response = await this.privateGetSpotV1MarginIsolatedPairs (params);
```